dev/breeze/pyproject.toml

```diff
@@ -59,8 +59,8 @@ dependencies = [
     "inputimeout>=1.0.4",
     "jinja2>=3.1.5",
     "jsonschema>=4.19.1",
-    "packaging>=23.2",
-    "prek>=0.1.2",
+    "packaging>=25.0",
```

Comment:
Here. You need to keep 23.2 for `v3-0-test` because of Python 3.9

Reply:
ah... i thought we need to update it as well. let's see how it works this time